src/frontend/src/optimizer/plan_node/logical_scan.rs

```diff
@@ -64,6 +65,7 @@ impl LogicalScan {
     pub fn create(
         table_name: String, // explain-only
         is_sys_table: bool,
+        is_cdc_table: bool,
```

Comment:
Having two boolean forming a product type is not a good practice. We may at least make it a sum type, i.e., `enum`.